### PR TITLE
[TEST] fill up alert group service test

### PR DESCRIPTION
**Pull request by abzymeinsjtu**
## Purpose of the pull request

add test code to cover all alert group implement

### CODE REVIEW

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/AlertGroupServiceTest.java [hunk 1]
```diff
@@ -19,9 +19,7 @@
 
 import static org.apache.dolphinscheduler.api.AssertionsHelper.assertDoesNotThrow;
 import static org.apache.dolphinscheduler.api.AssertionsHelper.assertThrowsServiceException;
-import static org.apache.dolphinscheduler.api.constants.ApiFuncIdentificationConstant.ALERT_GROUP_CREATE;
-import static org.apache.dolphinscheduler.api.constants.ApiFuncIdentificationConstant.ALERT_GROUP_DELETE;
-import static org.apache.dolphinscheduler.api.constants.ApiFuncIdentificationConstant.ALERT_GROUP_UPDATE;
+import static org.apache.dolphinscheduler.api.constants.ApiFuncIdentificationConstant.*;
```
Comment (SbloodyS) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/AlertGroupServiceTest.java]: Please avoid wildcard import.

**Review by SbloodyS — CHANGES_REQUESTED** [review]

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/AlertGroupServiceTest.java (hunk 1 above)
Comment (abzymeinsjtu) [reply dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/AlertGroupServiceTest.java]: updated

**Review by rickchengx — DISMISSED** [review]
LGTM

**Review by EricGao888 — DISMISSED** [review]

**Review by SbloodyS — DISMISSED** [review]
+1

**Review by rickchengx — APPROVED** [review]

**Pull request merged**